R/pkg/R/SQLContext.R

```diff
@@ -584,7 +584,7 @@ tableToDF <- function(tableName) {
 #'
 #' @param path The path of files to load
 #' @param source The name of external data source
-#' @param schema The data schema defined in structType
+#' @param schema The data schema defined in structType or a DDL-formatted string.
```

Comment:
is `DDL-formatted` a Spark specific thing? do we have a doc page on what that looks like?
perhaps we need to link to it?